app/src/main/java/org/wikipedia/page/PageTitle.kt

```diff
@@ -0,0 +1,212 @@
+package org.wikipedia.page
+
+import android.os.Parcelable
+import com.google.gson.annotations.SerializedName
+import kotlinx.parcelize.Parcelize
+import org.wikipedia.dataclient.WikiSite
+import org.wikipedia.language.AppLanguageLookUpTable
+import org.wikipedia.page.Namespace.Companion.fromLegacyString
+import org.wikipedia.settings.SiteInfoClient.getMainPageForLang
+import org.wikipedia.staticdata.TalkAliasData
+import org.wikipedia.staticdata.UserTalkAliasData
+import org.wikipedia.util.StringUtil.addUnderscores
+import org.wikipedia.util.StringUtil.removeNamespace
+import org.wikipedia.util.StringUtil.removeUnderscores
+import org.wikipedia.util.UriUtil.decodeURL
```

Comment:
Please import `org.wikipedia.util.StringUtil` and `org.wikipedia.util.UriUtil`.

Reply:
whoops